Octokit/Models/Response/ResourceRateLimit.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Diagnostics;
+using System.Globalization;
+
+using Octokit.Helpers;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class ResourceRateLimit
```

Comment:
I'm not sure where we ended up with this - is it `ResourceRateLimit` or `RateLimit`?